src/main/java/frc/robot/subsystems/Revolver.java

```diff
@@ -20,6 +21,8 @@
   private final CANSparkMax m_revolver;
   private int m_drumReps = 0;
 
+  private UnjamRevolver m_unjam = new UnjamRevolver(this);
```

Comment:
`final`, and move init to constructor for consistency

Reply:
resolved in f2e6a54